src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int firstSide;
+    private int secondSide;
+
+    public Rectangle(String color, String name) {

```

Comment:
The constructor of Rectangle should not generate random values. Instead, it should accept parameters for firstSide and secondSide.